src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4932,6 +4932,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="IDS_FeatureNameof" xml:space="preserve">
     <value>nameof operator</value>
   </data>
+  <data name="IDS_FeatureUnboundGenericTypesInNameof" xml:space="preserve">
+    <value>Unbound generic types in nameof operator</value>
```

Comment:
>Unbound [](http://example.com/codeflow?start=11&length=7)

Usually feature strings do not start with a capital letter

Reply:
sgtm.